peer/brontide.go

```diff
@@ -138,6 +138,23 @@ type TimestampedError struct {
 	Timestamp time.Time
 }
 
+// PeerDataStore is an interface representing the struture enabling the node
+// to carry out operations to store other peer's data.
+//
+//nolint:revive
```

Comment:
We don't want to disable all of revive, just the stutter rule firing. If you remove this line what is the linter output that is unhappy?

Reply:
![image](https://github.com/lightningnetwork/lnd/assets/59079323/bd5aae7d-9d08-4899-bd97-33e6ab25be1e)

Reply:
k yeah that's fine you can disable revive here.